spring-boot-project/spring-boot-tools/spring-boot-loader/src/main/java/org/springframework/boot/loader/jar/ZipInflaterInputStream.java

```diff
@@ -30,12 +30,15 @@
  */
 class ZipInflaterInputStream extends InflaterInputStream {
 
+	private final Inflater inflater;
+
 	private boolean extraBytesWritten;
 
 	private int available;
 
-	ZipInflaterInputStream(InputStream inputStream, int size) {
```

Comment:
I think we should keep this constructor and then call the new constructor proposed below with `this(inputStream, new Inflater(true), size)`;